src/components/App.tsx

```diff
@@ -1,33 +1,50 @@
 import React, { FC, memo } from 'react';
 import { HashRouter as Router, Route, Switch } from 'react-router-dom';
+import { Tab, TabList, TabPanel, Tabs } from 'react-tabs';
 
 import Header from 'components/Header';
 import PathwaysList from 'components/PathwaysList';
+import CriteriaList from 'components/CriteriaList';
 
 import BuilderRoute from './BuilderRoute';
 import ThemeProvider from './ThemeProvider';
 import { PathwayProvider } from './PathwayProvider';
 import { UserProvider } from './UserProvider';
+import { CriteriaProvider } from './CriteriaProvider';
 
 const App: FC = () => {
   return (
     <ThemeProvider theme="light">
       <UserProvider>
         <PathwayProvider>
-          <Router>
-            <Switch>
-              <Route path="/builder/:id/node/:nodeId">
-                <BuilderRoute />
-              </Route>
-              <Route path="/builder/:id">
-                <BuilderRoute />
-              </Route>
-              <Route path="/">
-                <Header />
-                <PathwaysList />
-              </Route>
-            </Switch>
-          </Router>
+          <CriteriaProvider>
+            <Router>
+              <Switch>
+                <Route path="/builder/:id/node/:nodeId">
+                  <BuilderRoute />
+                </Route>
+                <Route path="/builder/:id">
+                  <BuilderRoute />
+                </Route>
+                <Route path="/">
+                  <Header />
+                  <Tabs>
```

Comment:
It looks like the Tabs here could be broken out into their own component to help keep `App.tsx` manageable/readable while maintain separation of concerns and all that other good stuff.

It's not much, but in the long run it might save us some headache/refactoring.  Even if the other component is still defined in this same file.

I'm thinking about this block of code specifically:

```jsx
                  <Tabs>
                    <TabList>
                      <Tab>Pathways</Tab>
                      <Tab>Criteria</Tab>
                    </TabList>

                    <TabPanel>
                      <PathwaysList />
                    </TabPanel>
                    <TabPanel>
                      <CriteriaList />
                    </TabPanel>
                  </Tabs>
```